com.github.CineEncoder.yaml

```diff
@@ -0,0 +1,25 @@
+app-id: com.github.CineEncoder
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: CineEncoder
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=host
+
+modules:
+  - name: "Cine Encoder 2020"
+    buildsystem: qmake
+    build-commands:
+      - qmake
+      - install /app/share/applications/com.github.CineEncoder.desktop
+      - install /app/extra/export/share/icons/hicolor/64x64/apps/com.github.CineEncoder.png
```

Comment:
That's not correct. Either you specify a buildsystem or build commands. Also you don't specific those files as sources on your manifest. You should install them as post-install step instead.